docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -0,0 +1,184 @@
+---
+description: How to choose and connect to a testnet
+---
+
+# Connect to a Testnet
+
+The PegaSys Teku client allows you to connect a beacon-chain node to a public testnet and run validators on these networks. There are different ETH2 testnets to choose from.
+
+!!! note
+
+    Due to the fast pace of client and testnet development, this guide recommends connecting to the [Schlesi Testnet](https://github.com/goerli/schlesi). However, networks can experience stability issues and are prone to regular resets. It's recommended to regularly check the network's and client's documentation for updates.
+
+**Prerequisites**:
+
+* [Install Teku](/HowTo/Get-Started/Install-Binaries) Ethereum 2.0 client or [Build Teku from Source](/HowTo/Get-Started/Build-From-Source).
+
+Teku allows to select preconfigured networks with the `--network` flag:
+
+```bash
+  -n, --network=<NETWORK>    Represents which network to use.
+                               Default: minimal
+```
+
+Available options are: `minimal`, `mainnet`, `topaz`, and `schlesi`:
+
+* `minimal` is an ETH2 network configuration preset used for local testing and dev-nets.
+* `mainnet` is an ETH2 network configuration preset used for more dedicated test network setups targeting a future mainnet config.
+* `topaz` is a pre-configured single-client testnet maintained by the Prysmatic Labs team.
+* `schlesi` is a pre-configured multi-client testnet maintained by the Goerli Testnet Initiative.
+
+!!! tip
+
+    If you don't know which network to pick, choose `--network=schlesi`.
+
+## Connect to Schlesi Testnet
+
+* Name: `schlesi-v0-11`
+* Scope: multi-client testnet
+* ETH2 Spec version: `v0.11.2`
+* Teku version: `master`
+
+The Schlesi Testnet is a multi-client testnet used by developers and users of many different ETH2 clients. It implements version `v0.11.2` of the ETH2 specification and tries to be compatible with all ETH2 clients including Teku.
+
+Teku can connect a beacon-chain node to the Schlesi Testnet and synchronize the chain. Connecting Teku to Schlesi requires the latest `master` version of Teku.
+
+!!! note
+
+    Note, that building from `master` might come with certain instability and features may break without prior warning.
+
+Simply pass the `--network=schlesi` command-line flag to Teku.
```

Comment:
Remove the world "Simply" according to our writing guidelines please. Never consider something simple because it may be for you but not for the reader.